src/wx/wxvbam.cpp

```diff
@@ -772,6 +775,10 @@ void MainFrame::OnDropFile(wxDropFilesEvent& event)
     wxString* f = event.GetFiles();
     // ignore all but last
     wxGetApp().pending_load = f[event.GetNumberOfFiles() - 1];
+
+#ifndef NO_THREAD_MAINLOOP
+    GetPanel()->LoadGame(wxGetApp().pending_load);
+#endif
```

Comment:
@rkitover If I understood correctly, the IDLE event main loop keeps checking if this is set to load a game and start emulation. I would like to keep this idea away from the thread solution, if possible.